packages/go/openapi/src/paths/computers.computers.id.admin-rights.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 Specter Ops, Inc.
+#
+# Licensed under the Apache License, Version 2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+parameters:
+  - $ref: './../parameters/header.prefer.yaml'
+  - $ref: './../parameters/path.object-id.yaml'
+get:
+  operationId: GetComputerEntityAdminRights
+  summary: Get computer entity admin rights
+  description: Get a list, graph, or count of the systems this computer has admin rights to.
+  tags:
+    - Computers
+    - Community
+    - Enterprise
+  parameters:
+    - $ref: './../parameters/query.entity.skip.yaml'
```

Comment:
Opinion: we can omit the `./` here from the path since it should reference the current working directory by default.

By no means a blocker, looks good to me 😃